.gitignore

```diff
@@ -8,3 +8,8 @@
 /Release/
 /Examples/bin/
 *.tmp_proj
+src/.idea/.idea.src/riderModule.iml
```

Comment:
`src/.idea/` is enough to exclude everything.

Reply:
You are right :-)